packages/cli/src/helpers/test-helper.ts

```diff
@@ -0,0 +1,71 @@
+import { PrivateRunLocation, RunLocation } from '../services/abstract-check-runner'
+import { Session } from '../constructs'
+import type { Region } from '..'
+import { ReporterType } from '../reporters/reporter'
+import { isCI } from 'ci-info'
+
+const DEFAULT_REGION = 'eu-central-1'
```

Comment:
Why not export this and use it everywhere instead of having a hardcoded `DEFAULT_REGION` in all of these files:

```
packages/cli/src/commands/trigger.ts
packages/cli/src/commands/test.ts
packages/cli/src/commands/pw-test.ts
```